azure-iot-device/azure/iot/device/iothub/config.py

```diff
@@ -0,0 +1,15 @@
+import logging
+from azure.iot.device.common.config import PipelineConfig
+logger = logging.getLogger(__name__)
+
+class IoTHubPipelineConfig(PipelineConfig):
```

Comment:
>PipelineConfig [](start = 27, length = 14)

Not sure if we want the base class or not.  I like the idea of a place to store options that are common between DPS and IoTHub, but I'm not sure if a class hierarchy is overkill or not.  Right now, we have a single option that applies to both, and the other options (like keepalive interval, sas renewal intervals, and retry policy parameters) all apply to both.  So maybe we only need a base class and not the derived classes.  We can always add them later if we need them.

Reply:
ok that sounds fair